src/main/java/frc/robot/RobotContainer.java

```diff
@@ -32,18 +37,42 @@ public class RobotContainer {
   // The robot's subsystems
   private ADIS16470_IMU m_gyro = new ADIS16470_IMU();
   private final DriveSubsystem m_robotDrive = new DriveSubsystem(m_gyro);
-  private final Intake intake = new Intake();
-  private final Arm m_arm = new Arm();
-
+  private final Lock lockMode;
+  private final Camera camera = new Camera("slothEye");
```

Comment:
the camera is named "Slotheye" not "slothEye"

Reply:
Changed slothEye to Slotheye.